src/solve.jl

```diff
@@ -1,15 +1,11 @@
 function SciMLBase.__solve(prob::Union{SciMLBase.AbstractSteadyStateProblem,
                                        SciMLBase.AbstractNonlinearProblem},
                            alg::algType,
-                           reltol = 1e-3,
+                           args...;
+                           xtol = 0.,
```

Comment:
Why `xtol`?

Reply:
because it isn't a relative tolerance. It's an early exit if `norm(x-x_prev) < xtol`. We could call it `utol` if you prefer.

Reply:
(or `xabstol` if we in the future might add an `xreltol` to nonlinear solvers)

Reply:
why not? All that it means is that people won't be able to set the xtol, but they already weren't since it was being passed incorrectly.

Reply:
in that case, can I remove the ability (temporarily) to control the `xabstol` entirely for now? I think that calling this `reltol` is pretty dangerous since it makes it very easy to make the solve less strict than intended.

Reply:
I agree that mapping `reltol` to `xtol` is actively harmful and should be removed.  Holding off on adding a unified `xabstol` control is fine, but the base change of no longer linking `reltol` to `xtol` is important.